### ISSUE TITLE
Customer last name is encoded twice in the XML interface

### ISSUE THREAD

**Issue by nickshatilo** [magento/magento2#18361]
### Preconditions
1. Vanilla Magento 2.2.6 
2. PHP Version 7.0.31

### Steps to reproduce
1.  Create a customer via Backend with the last name "Foo & Bar Corp"
2. Create an order (via frontend is fine)
3. Export the order via XML interfcae

```
curl -XPOST -H 'Content-Type: application/json' http://hostname/rest/V1/integration/admin/token -d '{ "username": "admin_username", "password": "admin_password" }'
export B=code received above
 curl -X GET --header "Accept: application/xml" --header "Authorization: Bearer $B" "http://hostname/rest/english/V1/orders/1"
```

### Expected result
The `customer_lastname` tag should contain `Foo &amp; Bar Corp`

### Actual result
The `customer_lastname` tag contains `Foo &amp;amp Bar Corp`

### Summary
So I think at some point (2014) there was a bug in PHP (SimpleXML) that contained the issue with the ampersand, so that was a fix so that this won't influence any functionality. Currently, on 7.0+ this fix doesn't make sense and creates an error. 

### Proposed solution
`Magento\Framework\Webapi\Rest\Response\Renderer` class, `_formatValue` method. Delete the replacement of the ampersand. This solves the issue.

**Comment by magento-engcom-team** [comment]
Hi @nickshatilo. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@nickshatilo do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by nickshatilo** [comment]
@magento-engcom-team Yes, confirming that it reproduces on the vanilla Magento 2.2.6

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [x] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
@engcom-backlog-nazar Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-96219`, `MAGETWO-96220` were created

**Comment by magento-engcom-team** [comment]
Hi @novikor. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [ ] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [ ] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by magento-engcom-team** [comment]
Hi @nickshatilo. Thank you for your report. 
The issue has been fixed in magento/magento2#18380 by @nickshatilo in 2.3-develop branch
Related commit(s):
 - [b4181a0795e56c9954f6c03454af5e765727c597](https://github.com/magento/magento2/commit/b4181a0795e56c9954f6c03454af5e765727c597)
 - [491e91853826c75ba852732643b5679e5bff17ce](https://github.com/magento/magento2/commit/491e91853826c75ba852732643b5679e5bff17ce)

The fix will be available with the upcoming 2.3.1 release.

**Comment by magento-engcom-team** [comment]
Hi @nickshatilo. Thank you for your report. 
The issue has been fixed in magento/magento2#18362 by @nickshatilo in 2.2-develop branch
Related commit(s):
 - [ecd9dc42e56093a09193a13fac669cd0c3b1825c](https://github.com/magento/magento2/commit/ecd9dc42e56093a09193a13fac669cd0c3b1825c)
 - [0f75306061dcf6dc2942d6e165e8ba17f2ce3f6c](https://github.com/magento/magento2/commit/0f75306061dcf6dc2942d6e165e8ba17f2ce3f6c)
 - [ed497dc7fe6d704795ffbe01e78ae3015f782d7a](https://github.com/magento/magento2/commit/ed497dc7fe6d704795ffbe01e78ae3015f782d7a)

The fix will be available with the upcoming 2.2.8 release.
